zcbor/zcbor.py

```diff
@@ -2769,7 +2769,7 @@ def relativify(p):
                     Path("${CMAKE_CURRENT_LIST_DIR}") / path.relpath(Path(p), cmake_dir))
             except ValueError:
                 # On Windows, the above will fail if the paths are on different drives.
-                return Path(p).absolute()
+                return Path(p).absolute().as_posix()
```

Comment:
One question if the terminal in question doesn't support posix paths this will fail won't it? So in theory it should be 
```python
ap =  Path(p).absolute()
if os.type() == "windows":
  return ap.as_windows8)
else
  return ap.as_posix()
```

However I assume we are running this in zephyr's build system context which doesn't support running from `cmd` anyways.

Reply:
No, the point is actually to fix this on Windows :P. This function creates a Cmake file and Cmake uses the same path scheme (forward slash) everywhere, so we want forward slashes no matter what.